src/navigation/breadcrumbs/breadcrumbs.jsx

```diff
@@ -0,0 +1,291 @@
+    const borderRadiusMain = get(theme, 'shape.borderRadius.main');
+    const colorGrey500 = get(theme, 'palette.grey[500]');
+    const colorHighlight = get(theme, 'palette.cyan[500]');
+    const fontWeightBold = get(theme, 'typography.fontWeightBold');
+    const fontWeightMedium = get(theme, 'typography.fontWeightMedium');
+    const textColorPrimary = get(theme, 'palette.text.primary');
+    const textColorSecondary = get(theme, 'palette.text.secondary');
+    const transitionDurationShortest = get(theme, 'transitions.duration.shortest');
+
+    return {
+        root: {
+            overflow: 'hidden',
+            width: '100%',
+        },
+        tooltip: {
+            backgroundColor: colorGrey500,
+            borderRadius: borderRadiusMain,
+            fontSize: 12,
+        },
+        breadcrumbsList: {
+            overflow: 'hidden',
+            padding: 0,
+            textOverflow: 'ellipsis',
+            whiteSpace: 'nowrap',
+            margin: '-3px 0 -3px 0',
+        },
+        breadcrumb: {
+            color: textColorSecondary,
+            cursor: 'pointer',
+            display: 'inline',
+            padding: ({ showOnlyPreviousRoute }) => (showOnlyPreviousRoute ? '0 5px 0 0' : '0 5px 0 0'),
+            transition: `color ${transitionDurationShortest}ms`,
+            '& .navigation_breadcrumbs--breadcrumb_title > p': {
+                fontWeight: fontWeightMedium,
+                fontSize: 12,
+                color: textColorSecondary,
+            },
+            '& .icon-use-path': {
+                transition: `fill ${transitionDurationShortest}ms !important`,
+            },
+            '&:hover': {
+                '& .icon-use-path': {
+                    fill: colorHighlight,
+                },
+                '& .navigation_breadcrumbs--breadcrumb_title > p': {
+                    color: colorHighlight,
+                },
+            },
+        },
+        breadcrumbLast: {
+            '& .navigation_breadcrumbs--breadcrumb_title > p': {
+                fontWeight: fontWeightBold,
+                fontSize: 16,
+                color: textColorPrimary,
+            },
+            '&:hover': {
+                '& .navigation_breadcrumbs--breadcrumb_title > p': {
+                    color: textColorPrimary,
+                },
+            }
+        },
+        breadCrumbSeparator: {
+            display: 'inline',
+            padding: '0 5px 0 0',
+            '& .icon': {
+                marginRight: 5,
+            },
+        },
+        breadCrumbSeparatorTypography: {
+            display: 'inline',
+        },
+        breadcrumbTitle: {
+            display: 'inline',
+            transition: `color ${transitionDurationShortest}ms`,
+        },
+        breadcrumbTitleTypography: {
+            display: 'inline',
+        },
+    };
+});
+
+const BREADCRUMB_TITLE_MAX_LENGTH = 20;
+const ICON_TYPE_FIRST_BREADCRUMB = 'chevron-left';
+
+function Breadcrumbs(props) {
```

Comment:
Please add a component description here.

Reply:
Done

Reply:
Looks pretty good to me.  @morethanfire ?